reference/library/pom.xml

```diff
@@ -60,9 +60,175 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
```

Comment:
Would you be able to fix the indentation in this new section as it's a little all over the place.